src/test/java/com/aws/greengrass/util/platforms/unix/UnixPlatformTest.java

```diff
@@ -8,13 +8,16 @@
 
 import com.aws.greengrass.testcommons.testutilities.GGExtension;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
 import org.junit.jupiter.api.extension.ExtendWith;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.arrayContaining;
 import static org.hamcrest.Matchers.is;
 
 @ExtendWith({GGExtension.class})
+@EnabledOnOs({OS.LINUX, OS.MAC})
```

Comment:
you can just use disabledon windows instead

Reply:
Thanks for pointing this out.